rainforest/runner.go

```diff
@@ -94,8 +94,8 @@ func validateRerunParams(params RunParams) error {
 	if params.SiteID != 0 {
 		return errors.New("Site cannot be specified for rerun")
 	}
-	if params.Crowd != "" {
-		return errors.New("Crowd cannot be specified for rerun")
+	if params.ExecutionMethod != "" {
+		return errors.New("Execution method cannot be specified for rerun")
```

Comment:
```suggestion
		return errors.New("Execution Method cannot be specified for rerun")
```
Because "Execution Method" together is a "thing", like "Run Group" below.